examples/openvino/resnet50/README.md

```diff
@@ -0,0 +1,50 @@
+# ResNet50 Quantization
```

Comment:
We category examples by names. Can you move the examples to the examples folder instead? For example, you can add resnet files to `example/resnet/openvino/`